### PR TITLE
Speech bubble component

### PR DESCRIPTION
**Pull request by vkallerud**
🚀 Reusable speech bubble component. 
- ✨ _Additional changes worth mentioning: added an enum for custom colors (based on Figma file design), to ensure consistent colors across the game._

---
Need to make these accessibe in the templates/components the components is used: 
```
CustomColorsIO = CustomColorsIO;
PointerSide = PointerSide;
ArrowAlignment = ArrowAlignment;
```

_Example use in .html file:_
Screenshot 2024-07-08 at 14 52 38
Some parameters are optional.

### CODE REVIEW

**Review by ComputasAlex — APPROVED** [review]
This is a very cool component!
Should exist in some library for others to use.

Great job!

**Comment by vkallerud** [comment]
Waiting for 'Azure Static Web Apps CI/CD / Build and Deploy Job (pull_request)' to pass before merging (there seems to be too many other PRs open, so there are no threads available to run this one). Meanwhile I have written some short [documentation](https://computas.atlassian.net/wiki/spaces/S2/pages/679608327/Speech+bubble+component).

**Pull request merged**
